### PR TITLE
Issue #34

### PR DESCRIPTION
**Pull request by zoohair**

### CODE REVIEW

File: src/PGFPlots.jl [hunk 1]
```diff
@@ -147,34 +147,29 @@ type Axis
...
-            )
     Axis{P <: Plot}(plots::Vector{P};title=nothing, xlabel=nothing, ylabel=nothing, zlabel=nothing, xmin=nothing, xmax=nothing,
                     ymin=nothing, ymax=nothing, axisEqual=nothing, axisEqualImage=nothing, enlargelimits=nothing, axisOnTop=nothing, view="{0}{90}", width=nothing,
-					height=nothing, style=nothing, legendPos=nothing, xmode=nothing, ymode=nothing, colorbar=nothing, hideAxis=nothing, axisLines=nothing) =
+                    height=nothing, style=nothing, legendPos=nothing, xmode=nothing, ymode=nothing, colorbar=nothing, hideAxis=nothing, axisLines=nothing) =
         new(plots, title, xlabel, ylabel, zlabel, xmin, xmax, ymin, ymax, axisEqual, axisEqualImage, enlargelimits, axisOnTop, view, width, height, style, legendPos, xmode, ymode, colorbar, hideAxis, axisLines
             )
-
+            
+    Axis(;kwargs...) = Axis(Plot[]; kwargs...)
```
Comment (zoohair) [inline src/PGFPlots.jl]: I think this makes it cleaner than repeating all the keyword arguments for every single constructor.
I also made it possible to create axes without having to provide plots and to later push into them.

**Comment by zoohair** [comment]
@mykelk Let me know if you are ok with this being merged.
I can also add documentation for it if you think it's worth it.

**Comment by mykelk** [comment]
This looks good. But, should we have a type called Axes instead of Vector{Axis} for extensibility reasons?

**Comment by mykelk** [comment]
I think adding an example to the notebook would make this ready for a merge.

**Comment by zoohair** [comment]
The last build failed, but all I did was change the documentation.

Good practice would say not to merge if the build is failing, but from what I can see, master is also failing for the same reason (something about ccolors).
I'll merge this PR just to close the issue, and create a new one to track the ccolors issue.

**Pull request merged**